### PR TITLE
Remove container padding defaults

### PR DESCRIPTION
**Pull request by tomusborne**
This removes the 40px container padding default.

This one has definitely been filtered by users (https://community.generateblocks.com/t/change-the-default-container-padding/287) so it's important we check the implications of this on filtered defaults (different values or no value at all).

### CODE REVIEW

File: src/blocks/container/edit.js [hunk 1]
```diff
@@ -128,13 +129,28 @@ class GenerateBlockContainer extends Component {
 			}
 		}
 
-		// Update block version flag if it's out of date.
-		const blockVersion = 2;
+		// Set our old defaults as static values.
+		// @since 1.4.0.
+		if ( isBlockVersionLessThan( this.props.attributes.blockVersion, 2 ) ) {
```
Comment (tomusborne) [inline src/blocks/container/edit.js:134]: One issue here is that this returns true if it's a fresh new block as the `blockVersion` isn't set yet. This means new Containers (in this case) will add the old default as static values by default.

Comment (JeanPaiva) [reply src/blocks/container/edit.js:134]: @tomusborne adding `blockVersion` with a default value of two in the `container/attributes.js` would not solve this problem?

Comment (tomusborne) [reply src/blocks/container/edit.js:134]: Then the block would always use the default and never save a static version value. That would make it so we'd never know if a block had run any migration code (less than would never be true).

Comment (tomusborne) [reply src/blocks/container/edit.js]: This should fix it: https://github.com/tomusborne/generateblocks/pull/91/commits/89cd6ee9d94e0dcfd2b0903a50b13a8517592c85

**Review by JeanPaiva — APPROVED** [review]
All good.

**Pull request merged**